content/blog/2021-03-15-March-21-dvc-heartbeat.md

```diff
@@ -0,0 +1,197 @@
+﻿---
+title: March '21 Heartbeat
+date: 2021-03-15
+description: |
+  Monthly updates are here! News, use cases, videos, journal publications and more!
+descriptionLong: |
+  Monthly updates are here! Read all about our growing team, 
+  our CEO's interview on TFIR, Elle's talk at DataTalks.Club Conference and more!
+picture: 2021-03-15/mar21cover.png
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/february-21-heartbeat/669
+tags:
+  - Heartbeat
+  - CML
+  - DVC
+  - DataTalks.Club
+  - TFIR
+  - Queen's University
+  - arXiv
+  - MLOps
```

Comment:
Haven't reviewed blogs for a while but do we have all these tags? AFAIR we try to keep only a few tags (that already exist in previous blogs) unless it's very relevant to add more.